source/class/qx/tool/migration/M6_0_0.js

```diff
@@ -57,10 +57,10 @@ qx.Class.define("qx.tool.migration.M6_0_0", {
       ];
       // change names in .gitignore
       if ((await this.checkFilesToRename(migrateFiles)).length) {
-        let result = await this.renameFiles(migrateFiles);
+        await this.renameFiles(migrateFiles);
```

Comment:
This looks potentially dangerous. It calls `renameFiles` and subsequently looks at `dryRun`. Does `renameFiles` actually rename files? If so, maybe it should be inside the `else` block?

Reply:
`this.renameFiles` does its own check whether `dryRun` is enabled and renames only in this case. But you are right, this is not obvious from the method name. Not sure if I should rename it or revert this magic, so that one has to write a bit more code but it's more obvious what's going on.

Reply:
I renamed the method to make more obvious what they do.